marimo/_server/api/endpoints/editing.py

```diff
@@ -77,3 +78,13 @@ async def stdin(request: Request) -> BaseResponse:
     app_state.require_current_session().put_input(body.text)
 
     return SuccessResponse()
+
+
+@router.post("/install_missing_packages")
+@requires("edit")
```

Comment:
should we do this on `read` too when in pyodide?

Reply:
added a TODO